ci-operator/config/openshift/tektoncd-catalog/openshift-tektoncd-catalog-release-next.yaml

```diff
@@ -11,25 +11,12 @@ base_images:
     name: '4.1'
     namespace: ocp
     tag: base
-  cli:
-    name: 'origin-cli'
-    namespace: tektoncd
-    tag: latest
 
 build_root:
   project_image:
     dockerfile_path: openshift/Dockerfile
 
 images:
```

Comment:
should w either remote `images:` of make it `images: null`, or `images: {}`

Reply:
however, it looks like the CI here is happy with how it is

Reply:
We can, however, we need it for https://github.com/openshift/release/pull/5504/files#diff-a9ecb60fc9f964ac7d3fe4459936aaf2R20-R27 (I mean still there is one image job needs to build)?